docs/source/release/v6.9.0/Diffraction/Engineering/Bugfixes/34368.rst

```diff
@@ -0,0 +1 @@
+- Avoid writing invalid background estimation parameters in the Engineering Diffration's table of fitting tab
```

Comment:
I think this could be clarified and references to the docs added.
Something along the lines of this?
```
- When invalid parameters are entered for the background estimation in the :ref:`Fitting tab <ui engineering fitting>` of the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` they are overwritten with the last valid parameters used when the background is next calculated (immediately if `Subtract BG` is checked in the table).
```